cs/src/core/Index/FASTER/FASTERImpl.cs

```diff
@@ -1439,6 +1509,33 @@ private void BlockAllocate(int recordSize, out long logicalAddress)
             }
         }
 
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
>[MethodImpl(MethodImplOptions.AggressiveInlining)] [](start = 8, length = 50)

Can compiler inline a recursive method?

Reply:
Since this is tail recursion, it should get inlined (I think).